modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -107,15 +107,15 @@
     "xml" :fhir+xml
     nil))
 
-(defn- request-format
+(defn- accept-header-format
```

Comment:
This is not only about the `Accept` header. Also the `_format` query param is considered. That's why I called the function `request-format`.